tests/unit/test_tools_archive.py

```diff
@@ -5,17 +5,15 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 """Tests for webcompat.com archive tools.
 
-TODO: test for json to html
```

Comment:
This a cool way to develop locally, and plan out commits ahead of time (something I struggle with a lot).

Reply:
I also starts to do things like this https://github.com/webcompat/webcompat.com/blob/034d7d7c68bf3246f3d4afd60a75839a8683d3bc/tests/unit/test_tools_archive.py#L140-L143

aka creating a test that is marked as skipped. 

```
(env) ~/code/webcompat.com-karlcow % COLUMNS=50 pytest -r s tests/unit/test_tools_archive.py
============== test session starts ===============
platform darwin -- Python 3.8.5, pytest-5.4.3, py-1.8.1, pluggy-0.13.1
rootdir: /Users/karl/code/webcompat.com-karlcow, inifile: pytest.ini
plugins: mock-3.2.0
collected 13 items                               

tests/unit/test_tools_archive.py ......... [ 69%]
.sss                                       [100%]

============ short test summary info =============
SKIPPED [1] tests/unit/test_tools_archive.py:136: TODO: HTTP ERROR log for comments fetching
SKIPPED [1] tests/unit/test_tools_archive.py:142: TODO: JSON for ALL comments
SKIPPED [1] tests/unit/test_tools_archive.py:148: TODO: Test comment as list of Comment
========= 10 passed, 3 skipped in 0.24s ==========

```